src/lib.ts

```diff
@@ -108,8 +145,31 @@ export const args = Args.create("railo", "A script for farming elf stuff", {
     ],
     default: "both",
   }),
+  orb: Args.string({
+    options: [
+      ...[...Object.values(trainbots)]
+        .map((o) => Object.keys(o))
+        .flat()
+        .map((key) => [key, `Target the ${key} trainbot with the orb.`] as [string, string]),
+      ["none", "Don't use it!"],
+    ],
+    default: "none",
+  }),
 });
 
+let orbTarget: Monster | null = null;
+export function validateOrbArgument(target: string, car: string) {
```

Comment:
In a world where you decided to allow multiple targets in a given train you could have this via a comma separated target list you could either:

1) make target a `string[]` and `.split(',')` in the calling context.
2) keep target a `string` and `.split(',')` it here.